### PR TITLE
Add pulpcore tag, add support for merge mode

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

**Comment by pcreech** [comment]
@evgeni @ehelms this could probably use some bikeshedding over the name

File: configs/pulpcore/nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+:project: pulpcore
+:github_org: pulp
```
Comment (evgeni) [inline configs/pulpcore/nightly.yaml:3]: not sure what implications this has, but pretty sure it won't live in that org ;-)

File: configs/pulpcore/nightly.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    arches:
+      - x86_64
+    inherits:
+      pulpcore-nightly-el8-build:
+        pulpcore-nightly-el8-override: 0
+      pulpcore-nightly-el8-override:
+        pulpcore-nightly-el8: 0
+    external_repos:
+      - name: centos8-baseos
+        mode: bare
```
Comment (evgeni) [inline configs/pulpcore/nightly.yaml:23]: what is bare, and are we intentionally not using the "flattened" repo?

Comment (pcreech) [reply configs/pulpcore/nightly.yaml:23]: 'bare' is a mode added to koji to support modular repositories.  It tells koji to call mergerepo_c with --pkgorigins, which allows normal mergerepo behaviour, while also providing pkgorigins file for koji.

Comment (pcreech) [reply configs/pulpcore/nightly.yaml:23]: As far as which repo we're using, I structured this around the existing pulpcore tag, which is utilizing the individual repos.  We never converted over to the flattened repo for pulpcore, and we haven't had an issue with it (yet)

File: configs/pulpcore/nightly.yaml (hunk 1 above)
Comment (pcreech) [reply configs/pulpcore/nightly.yaml:3]: ah, yeah, :D old habits

**Pull request merged**